src/pages/Home/components/HomeView/index.js

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types";
+import Menu from "../../../../components/menu";
+
+const HomeView = () => {
+  return (
+    <div>
+      HOME PAGE
+      <Menu></Menu>
```

Comment:
When you have no content inside react component use it in self-closing way. `<Menu />`